fleasite/settings_release.py

```diff
@@ -6,15 +6,17 @@
 SECRET_KEY = "django-insecure-vb_08&&o(0e9_o&#^ucukz091h6d0o7eiw01$0gsm&&v&kdmd5"
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = False
+DEBUG = True
```

Comment:
I don't really think we need to put `Debug = True` on a prod server but I can feel your difficulties during development. The change from #22 should resolve the problem here. I've setup the environment variables on AWS and Azure so they can continue to run the prod version of code. Please kindly revert the changes to `settings_*.py` files